.github/workflows/qpext-docker-publish.yml

```diff
@@ -74,9 +76,14 @@ jobs:
 
           # Use Docker `latest` tag convention
           [ "$VERSION" == "master" ] && VERSION=latest
+          [[ "$VERSION" =~ ^release- ]] && VERSION=$(echo $VERSION | sed 's/^release-//')-latest
```

Comment:
so 0.11 release branch will use `0.11-latest` ? 
I think retag it to 2 tags:  `0.11-$COMMIT_SHA` and `0.11` . then quay.io will show these 2 images are the same. 
WDYT?

Reply:
> so 0.11 release branch will use 0.11-latest ?

Yes.

> I think retag it to 2 tags: 0.11-$COMMIT_SHA and 0.11 . then quay.io will show these 2 images are the same.

Would it be useful? I don't see a reason for accumulating images.

Reply:
The image manifest on quay should include git ref if we don't include it in the image tag itself, but it's definitely easier for lookup purposes to do commit sha

Reply:
> The image manifest on quay should include git ref if we don't include it in the image tag itself, but it's definitely easier for lookup purposes to do commit sha

The `-latest` suffix is the typical docker nomenclature. I'd think it maps "naturally". The `v0.11-latest` tag would map to `release-v0.11` git ref.

If we strictly want a git-ref, I can create both the git-ref tag and the docker-friendly tag: `release-v0.11` and `v0.11-latest`. 

But, then, I'm still not sure about the tag carrying the SHA, as that would mean accumulating images.

Reply:
I'm still thinking on the tag with the SHA.

Do we *really* need it?